spec/services/monthly_district_data_service_spec.rb

```diff
@@ -2,13 +2,18 @@
 
 RSpec.describe MonthlyDistrictDataService, reporting_spec: true do
   around do |example|
-    freeze_time_for_reporting_specs(example)
+    # This is in the style of ReportingHelpers::freeze_time_for_reporting_specs.
+    # Since FacilityAppointmentScheduledDays only keeps the last 6 months of data, the date cannot be a
+    # fixed point in time like the spec helper.
+    Timecop.freeze("#{Date.today.end_of_month} 23:00 IST") do
+      example.run
+    end
   end
 
   let(:organization) { FactoryBot.create(:organization) }
   let(:facility_group) { create(:facility_group, organization: organization) }
-  let(:facility1) { create(:facility, facility_group: facility_group, facility_size: :community) }
-  let(:facility2) { create(:facility, facility_group: facility_group, facility_size: :community) }
+  let(:facility1) { create(:facility, block: "Eagle Estates", facility_group: facility_group, facility_size: :community) }
+  let(:facility2) { create(:facility, block: "Summer Heights", facility_group: facility_group, facility_size: :community) }
```

Comment:
The order in which facilities appear in a csv can be predicted by giving a specific `block name` to each facility. Specs can be written accurately since the facilities appear in alphabetic order.

Reply:
Can name the blocks: `Block 1 - alphabetically first` and the other one `Block 2 - alphabetically second` to make this explicit.